### PR TITLE
Actually use the default plotter mode from config without overwrite

### PR DESCRIPTION
**Pull request by matthewware**
Open request for comments.  Looks like everyone is just manually setting single_plotter_mode to true in the QubitExpFactory.  This is confusing dinosaurs like myself who thought the config.py settings were being respected.  Do others have strong feelings on how this should work?  Seems you could easily set a global variable if that's how you always wanted things to behave and overwrite as necessary in the run function.

### CODE REVIEW

File: src/auspex/exp_factory.py [hunk 1]
```diff
@@ -153,18 +153,26 @@ class QubitExpFactory(object):
...
+                                    repeats=repeats)
         exp.run_sweeps()
+
+        # set the plotting mode back to the default value
+        auspex.config.single_plotter_mode = spm
         return exp
 
     @staticmethod
     def create(meta_file=None, meas_file=None, expname=None, calibration=False, save_data=True,
-               cw_mode=False, instr_filter=None, repeats=None, single_plotter=True):
```
Comment (grahamrow) [inline src/auspex/exp_factory.py]: This way the convenience method doesn't work if you create the experiment and subsequently add loops.
```python
exp = exp_factory.create()
exp.add_sweep(...)
exp.run_sweeps()
```
Is that okay?

Comment (matthewware) [reply src/auspex/exp_factory.py]: That or have `run_sweeps` coded like `run` with a save settings, update, run, load saved settings?

**Comment by matthewware** [comment]
We could write a `single_plot` function that just updated the config.py

**Comment by dieris** [comment]
I'm fine with the global setting as default. But I'd prefer to keep the flexibility of setting single_plotter also at the exp.create level

**Comment by gribeill** [comment]
I agree with @dieris

**Comment by matthewware** [comment]
The best way I can come up with to do this is by passing the config on the the experiment object since create dosen't actually run anything we'd need the context to carry through to the `run_sweeps` function.  I haven't tested the above commits but I'll try to get to them this afternoon, Utah permitting.

**Comment by matthewware** [comment]
Issue is now ancient history and can be deleted as plotting is done without matplotlib server or bokeh.

**Pull request closed without merging**